src/fundus/publishers/de/motorsport_magazin.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class MotorSportMagazinParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector: CSSSelector = CSSSelector("section.article-body")
+        _subheadline_selector: CSSSelector = CSSSelector("p.teaser")
```

Comment:
I see the potential for the misunderstanding, but in Fundus this part of the article is the summary, so it should be the _summary_selector (and also updated within the body() function)
But then this leaves a space for the actual subheadline selector. This article is a good example for that: https://www.motorsport-magazin.com/formel1/news-288319-nico-huelkenberg-wechselt-zu-sauber-audi-sichert-sich-deutsche-formel-1-speerspitze/ . I would suggest something along the lines of `section.article-body > h2`

Reply:
added!